dwhdelft.nl/pages/mixup.vue

```diff
@@ -0,0 +1,212 @@
+      image: events-dancenights
+    - title: Parties
+      description: Disco, hard rock, Oktoberfest, Halloween and more... There’s always time for a party.
+      image: events-parties
+    - title: Drag Shows
+      description: Our own drag house HAUS of 4D performs every two months with amazing shows.
+      image: events-dragshow
+    - title: Events
+      description: Karaoke, tastings, vinyl night or a talent show, there is a lot to do on a night at MIXUP.
+      image: events-silentdisco
+  highlights:
+    title: Our Highlights
+    events:
+      - title: MIXUP Origin
+        description: The first ever party under the MIXUP branding was MIXUP Origin, the opening party! Miss Abby OMG gave a wonderful performance and people were living!
+        image: highlight-abbyomg
+      - title: Summer Party
+        description: The weather was hot and the people even hotter! DJ Maarten brought us tropical tunes whilst we could sip summer cocktails and cool off in an actual swimming pool!
+        image: highlight-summer-party
+
+nl:
+  title: MIXUP
+  events: Onze UPkomende evenementen
+  intro:
+    - MIXUP is dé queer uitgaansavond van Delft. Iedere zaterdagavond zijn we vanaf 22:00 open en bijna
+      elke week zijn we meer dan alleen een bar. Drag performances van HAUS of 4D, de beste karaoke van de stad,
+      heerlijke proeverijen, MIXUP Alternative met bands, dansavonden met hitjes van nu of juist je favoriete
+      klassiekers, een hitsig Halloween en andere themafeesten, Eurovisie watch parties en meer! En als we dan toch
+      eens gewoon een bar zijn, is het heel gezellig bijkletsen en misschien een dansje wagen.
+    - MIXUP wordt volledig mogelijk gemaakt door een team enthousiaste vrijwilligers van DWH. Ben je enthousiast geworden en wil je
+      bijdragen aan en onderdeel worden van een fantastische community? Wordt dan lid.
+    - Bang om alleen te komen? We kunnen je aan een barbuddy koppelen om te leren kennen, die je ook aan anderen kan voorstellen!
+  invite:
+    announcement: 'Kom gerust langs op onze baravond:'
+    time: Elke Zaterdag vanaf {0}
+  membership_button: Schrijf je nu in bij DWH
+  barbuddy_button: Vind een buddy
+  instagram: Onze evenementen en UPdates
+  bulletPoints:
+    - title: Dansavonden
+      description: Elke week komen we samen voor drankjes, vaak is er zelfs een DJ om die voetjes op te bewegen!
+      image: events-dancenights
+    - title: Feesten
+      description: Disco, hard rock, Oktoberfest, Halloween en nog veel meer... Er is altijd tijd voor een feestje.
+      image: events-parties
+    - title: Drag optredens
+      description: Ons eigen drag huis HAUS of 4D geeft elke twee maanden geweldige shows
+      image: events-dragshow
+    - title: Evenementen
+      description: Karaoke, proeverijen, vinyl nachten of een talentenshow, er is veel te beleven op een MIXUP avond.
+      image: events-silentdisco
+  highlights:
```

Comment:
doesn't have to be fixed in this PR, but content wise, this feels pretty "light". Only the origin party of a couple years ago and one other party

Reply:
Agree, but if we have something in place, it can be filled up later. This is for MIXUP to do.